src/actions/PathAction.js

```diff
@@ -0,0 +1,51 @@
+import { push } from 'react-router-redux';
```

Comment:
note: we should probably move away from using push since the repo is archived and doesn't seem necessary if you use react router + redux